src/upsample.jl

```diff
@@ -134,8 +134,17 @@ function upsample_linear(x::AbstractArray{T,<:Any}; size) where T<:Integer
     return round.(T, res)
 end
 
+# compatibility layer for old versions of NNlibCUDA
+# old versions overload upsample_linear_wcn, new versions overload upsample_linear_kernel
```

Comment:
Maybe say something like `can be removed from NNlib 0.9, i.e. revert https://github.com/FluxML/NNlib.jl/pull/414` in the comment?